setup.py

```diff
@@ -11,7 +11,9 @@
       long_description="",
       url="",
       packages=['opencivicdata.divisions'],
+      namespace_packages=['opencivicdata'],
       include_package_data=True,
+      install_requires=['backports.csv'],
```

Comment:
https://hynek.me/articles/conditional-python-dependencies/ has a way to do this for Python 3 only